test/integration/targets/connection_ssh/runme.sh

```diff
@@ -22,20 +22,22 @@ if command -v sshpass > /dev/null; then
             -e ansible_sshpass_prompt=notThis: \
             -e ansible_password=foo \
             -e ansible_user=definitelynotroot \
-	    -i test_connection.inventory \
+        -i test_connection.inventory \
```

Comment:
It looks like the indentation is off here:

```suggestion
            -i test_connection.inventory \
```